### PR TITLE
Add alpha 4 changelog.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

**Comment by vpetrovykh** [comment]
I can see a whole bunch of changes to server logging, but I'm not sure if we want to include it in the changelog yet. My understanding so far was that this was mainly for our own use and not any sort of "API" intended for others, at least at this point.

File: docs/changelog/1_0_a4.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  the default EdgeDB CLI. Check out the details in this `RFC 1001
+  <edbserver_>`_.
+
+
+Server
+======
+
+* Remove reliance on Postgres table inheritance (:eql:gh:`#1468`)
+* Fix handling of ROLE commands on RDS (:eql:gh:`#1472`).
+* Make datadir compatibility check earlier (:eql:gh:`#1481`).
```
Comment (1st1) [inline docs/changelog/1_0_a4.rst:46]: No need for this one

File: docs/changelog/1_0_a4.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* We have a new ``edgedb server`` group of commands that ships with
+  the default EdgeDB CLI. Check out the details in this `RFC 1001
+  <edbserver_>`_.
+
+
+Server
+======
+
+* Remove reliance on Postgres table inheritance (:eql:gh:`#1468`)
+* Fix handling of ROLE commands on RDS (:eql:gh:`#1472`).
```
Comment (1st1) [inline docs/changelog/1_0_a4.rst:45]: no need for this one

File: docs/changelog/1_0_a4.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Make datadir compatibility check earlier (:eql:gh:`#1481`).
+* The ``edgedb-server`` installed by official installers is now called
+  ``edgedb-server-1-alpha4``.
```
Comment (1st1) [inline docs/changelog/1_0_a4.rst:48]: This needs to be expanded recommending `edgedb server start` cc @tailhook

File: docs/changelog/1_0_a4.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+Misc
+====
+
+* Rename ``fetch`` to ``query`` in the `edgedb-js
+  <https://github.com/edgedb/edgedb-js>`_ and the `edgedb-python
+  <https://github.com/edgedb/edgedb-python>`_ drivers.
+* Add connection pooling to `edgedb-js <https://github.com/edgedb/edgedb-js>`_
+  driver.
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.7.4.
```
Comment (1st1) [inline docs/changelog/1_0_a4.rst:60]: to v0.8

File: docs/changelog/1_0_a4.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
+===========
+1.0 Alpha 4
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 4.
```
Comment (1st1) [inline docs/changelog/1_0_a4.rst:6]: Barnard's Star

Comment (vpetrovykh) [reply docs/changelog/1_0_a4.rst:6]: OK

File: docs/changelog/1_0_a4.rst (hunk 4 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a4.rst:60]: Updated.

File: docs/changelog/1_0_a4.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a4.rst:45]: Removed Postgres reference.

Comment (vpetrovykh) [reply docs/changelog/1_0_a4.rst:45]: Removed `ROLE` reference.

File: docs/changelog/1_0_a4.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a4.rst:46]: Removed datadir reference.

**Comment by 1st1** [comment]
@vpetrovykh Please fix issues found by `edb test -k cqa`

**Comment by vpetrovykh** [comment]
I've fixed the `cqa` issues.

**Comment by vpetrovykh** [comment]
I've also added the transaction marker feature to the CLI changelog section.

**Pull request merged**